tests/foreman/api/test_hostcollection.py

```diff
@@ -356,6 +357,7 @@ def test_positive_update_host(self):
         host_collection = host_collection.update(['host'])
         self.assertEqual(host_collection.host[0].id, self.hosts[1].id)
 
+    @upgrade
```

Comment:
Sort Decorators alphabetically in this and all further files wherever applicable.